JeffKersting/chatterBox-api
# Feature initialize database

## *Purpose:*

Sets up the initial backend file structure, with dependencies and configuration file.

*What does this merge do? Check all that apply.*

## *Description: What does this PR do?*

Initializes backend.

*Briefly, but clearly describe what you did, and explain any new code.*

Adds dependencies and backend configuration.

## *Instructions for reviewer:*

Review index.js and related config files.

*Where should the reviewer start?*

Index.js

*How should this be manually tested?*

Running the FE react app and the BE server, ensure connection is made.

*Any background context you want to provide?*

Will need to build the postgres database.

## Checklist:
